### PR TITLE
updated viscosity (1.5.11)

### PR DESCRIPTION
**Pull request by vitorgalvao**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Merged as e662fb1192f148acb165cbb5a56644f847cbf210.

File: Casks/viscosity.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 cask 'viscosity' do
-  version :latest
-  sha256 :no_check
+  version '1.5.11'
+  sha256 'f235d2db655151d15976beee293bc2f6bc85025fdfe83c1b015e0d413547ba95'
 
   url 'https://www.sparklabs.com/downloads/Viscosity.dmg'
-  appcast 'http://www.viscosityvpn.com/update/viscosity.xml',
-          :checkpoint => '1dba070850ab55b9c94b8287e3f67a167c502ccf4da6a84088cb38fb0278e571'
```
Comment (tsparber) [inline Casks/viscosity.rb]: Appcast seems to work for me and contains 1.5.11. Why is it removed?

**Comment by jnunemaker** [comment]
I'm getting an error installing since this was merged.

```
Error: Cask 'viscosity' definition is invalid: Bad header line: parse failed
```

Any ideas on how to trouble shoot or fix?

**Comment by adidalal** [comment]
@jnunemaker 

This error is not a bug, it's caused by an update to how `homebrew-cask` works. Details can be found [here](https://github.com/caskroom/homebrew-cask#important-december-update-homebrew-cask-will-now-be-kept-up-to-date-together-with-homebrew-see-15381-for-details-if-you-havent-yet-run-brew-uninstall---force-brew-cask-brew-update-to-switch-to-the-new-system).

To fix the issue, please run:
```bash
brew update; brew cleanup; brew cask cleanup
brew uninstall --force brew-cask; brew update
```

If you are still having problems, [open an issue with the appropriate template](https://github.com/caskroom/homebrew-cask#questions-wanna-chat)

**Comment by jnunemaker** [comment]
That worked.

Seems like there should be a more helpful error message when running the command that says something about an update being needed or something. I'm not sure that I would ever think look at the README of cask when I have a compile issue on an unrelated thing.

**Comment by jnunemaker** [comment]
Also, thanks! Sorry I forgot to say that. Appreciate the help.

**Comment by vitorgalvao** [comment]
> Seems like there should be a more helpful error message when running the command that says something about an update being needed or something.

https://github.com/caskroom/homebrew-cask/issues/16344#issuecomment-168836594.

**Comment by jawshooah** [comment]
> Seems like there should be a more helpful error message when running the command that says something about an update being needed or something. I'm not sure that I would ever think look at the README of cask when I have a compile issue on an unrelated thing.

There is no way for us to go back in time and add a more detailed error message for an older version.

**Comment by jawshooah** [comment]
@vitorgalvao beat me to the punch!

**Pull request closed without merging**